docs/python-examples/fixtures/conftest.py

```diff
@@ -0,0 +1,90 @@
+import logging
+
+import pytest
+
+import ocs
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='class')
+def storage_class(request):
+    """
+    Storage class fixture
+
+    Returns:
+        StorageClass: object of storage class
+    """
+    def fin():
+            sc.delete()
```

Comment:
is this using a `TAB` or 8 spaces? looks odd here and on other `fin()` calls

Reply:
There was one line before sc.delete with if condition which was deleted by @vasukulkarni here: https://github.com/red-hat-storage/ocs-ci/commit/c731450a2872cee90a11843d574a2775d872b05b#diff-ae168d5153b97475ccfbc0d0a3d6ad76L20